crates/formats/src/param.rs

```diff
@@ -551,6 +556,38 @@ impl<'a, T: traits::ParamFileLayout> ParamCommon<'a> for Param<'a, T> {
             .binary_search_by_key(&row_id, |rd| rd.id.into())
             .ok()
     }
+
+    fn dyn_rows(&self) -> Box<dyn Iterator<Item = UnifiedRowInfo<'_>> + '_> {
```

Comment:
I think `Untyped` would be better than `Unified`

Reply:
Done.